concepts/enums/introduction.md

```diff
@@ -0,0 +1,80 @@
+# Introduction
+
+## Enumerations
+
+So far we have mostly handled numbers and strings.
+To model the real world, we might want a limited number of options that a variable can take on.
+If you work for a skateboard manufacturer the deck material can be selected from maple, bamboo, or plastic.
+
+You could use integers to encode those values, but would have to use extra code to check if there is an invalid value coming from the system for the material.
```

Comment:
I'm not sure if enums can really prevent invalid values since integers can be converted to enums without any kind of check.  
I'd rather argue that enums allow you to use expressive symbols (`maple`) instead of magic values (`0`), and they prevent mixing different types by accident.  
But that might be semantics.

Reply:
What do you think of this change?

Reply:
Okay, another version?